client/src/components/Share/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  FacebookIcon,
+  WhatsappIcon,
+  EmailIcon,
+  TwitterIcon,
+  EmailShareButton,
+  FacebookShareButton,
+  TwitterShareButton,
+  WhatsappShareButton,
+} from 'react-share';
+
+import { Container, Icons, Text } from './style';
+
+const Share = ({ text, facebook, twitter, email, whatsapp }) => {
```

Comment:
you can give the text a defualt value
```suggestion
const Share = ({ text = 'Share this page', facebook, twitter, email, whatsapp }) => {
```

in this way, you don't have to give it that same value each time you use this component, and you can just overwrite it if you want a different text